pkg/server/sandbox_run.go

```diff
@@ -452,6 +452,14 @@ func (c *criService) generateSandboxContainerSpec(id string, config *runtime.Pod
 	}
 	g.SetProcessOOMScoreAdj(adj)
 
+	for _, r := range c.config.RuntimeOptionsPodAnnotations {
```

Comment:
Can we support regular expression match? As suggested in https://github.com/containerd/cri/pull/1077#issuecomment-470626651

Reply:
done